go.mod

```diff
@@ -15,12 +15,14 @@ replace github.com/spf13/cobra => github.com/spf13/cobra v1.4.0
 // to avoid 'sonatype-2020-1055' non-CVE vulnerability introduced by github.com/go-ldap/ldap/v3@v3.1.10
 replace github.com/go-ldap/ldap/v3 v3.1.10 => github.com/go-ldap/ldap/v3 v3.4.3
 
+replace github.com/prometheus/client_golang => github.com/prometheus/client_golang v1.11.1
```

Comment:
Could we add a short comment explaining why this is required, in line with the other replace directives?

Also, should we use the latest `1.13.0` version?